packages/block-editor/src/components/block-types-list/index.js

```diff
@@ -23,7 +29,12 @@ function BlockTypesList( {
 		 * Safari+VoiceOver won't announce the list otherwise.
 		 */
 		/* eslint-disable jsx-a11y/no-redundant-roles */
-		<ul role="list" className="block-editor-block-types-list">
+		<Composite
+			as="ul"
+			role="list"
```

Comment:
`list` is not a valid composite role. Screen reader users won't have a hint that they should use arrow keys to navigate through the items, so this would be probably less accessible for some of them. But we can use `listbox` here.

```suggestion
			role="listbox"
```

Also, this components requires either `aria-label` or `aria-labelledby` prop. Since the category title element is already in the document, the best option is to pass its ID to the `aria-labelledby` prop here.

Reply:
I've used an aria-label as the alternative would have required a bigger refactor (separate components)